### DISCUSSION TITLE
Use merge queue

### DISCUSSION THREAD

**Discussion by turol** [assimp/assimp#5180] [Ideas]
GitHub just added a merge queue feature:

https://github.blog/2023-07-12-github-merge-queue-is-generally-available/

Let's use it.

**Comment by turol** [comment]
@kimkulling Thoughts? Should make your work a lot easier.

**Comment by kimkulling** [comment]
Is this just for Enterprise users?

**Comment by kimkulling** [comment]
Sounds interesting to me. It seems to be a good way to avoid our blocked merges.
